### ISSUE TITLE
DB Backups Broken

### ISSUE THREAD

**Issue by CHTJonas** [camdram/camdram#407]
Database backups are now failing with the following error message:
```mysqldump: unknown variable 'login-path=backup'```

**Comment by CHTJonas** [comment]
I notice from `/var/log/apt/history.log` that MySQL on Tiberius was recently downgraded from 5.6.39 to 5.5.60. My guess is that this caused us to lose support for the `--login-path=backup` option on the command line.

**Comment by hoyes** [comment]
I need to get better at talking about these changes now there are more people looking...

Prior to upgrading to Debian Jessie we were using https://www.dotdeb.org/ to give us PHP 5.6, which came with the side effect of bringing in MySQL 5.6. Upon upgrading I dropped dotdeb as Jessie has PHP 5.6 natively but ended up leaving MySQL at 5.6 at the time because downgrading the DB is a bit of a 'mare. It alll kinda worked but we were getting a spew of PHP warnings in production because the PHP client was 5.5 talking to a 5.6 server.

I finally deleted MySQL 5.6 on Saturday, replaced with MySQL 5.5 from the Jessie repos and rebuilt the database. Looks like Stretch sticks on 5.5 too so we're probably stuck with that for a while unless there's a strong reason to pull in a newer version from somewhere.
